rosys/vision/camera/camera.py

```diff
@@ -20,16 +20,15 @@ def __init__(self,
                  id: str,  # pylint: disable=redefined-builtin
                  name: Optional[str] = None,
                  connect_after_init: bool = True,
-                 streaming: bool = True,
-                 image_grab_interval: float = 0.1,
+                 streaming: bool = False,
```

Comment:
Wow. @NiklasNeugebauer, @falkoschindler this breaking change took me some time. I wondered why my camera is not delivering any images anymore.

Reply:
Oh, that was unintentional. 
I will try to explicitly list all breaking changes from on, which will hopefully help me catch that.